Sources/PrimerSDK/Classes/User Interface/Root/PrimerCardFormViewController.swift

```diff
@@ -17,6 +17,19 @@ class PrimerCardFormViewController: PrimerFormViewController {
     private let cardholderNameContainerView = PrimerCustomFieldView()
     private let submitButton = PrimerOldButton()
     
+    // todo: refactor to dynamic form builder
+    private lazy var expiryAndCvvRow = row
```

Comment:
Is there any reason for those two to be properties? Why not move them in the function scope?